src/Core/Addon/Theme/Theme.php

```diff
@@ -210,9 +217,28 @@ public function getLayoutNameForPage($page)
         return $layout_name;
     }
 
+    /**
+     * Returns full path for provided page identifier
+     *
+     * @param string $page page identifier
+     *
+     * @return string layout full path
+     */
     public function getLayoutRelativePathForPage($page)
     {
-        return 'layouts/' . $this->getLayoutNameForPage($page) . '.tpl';
+        return $this->getLayoutRelativePathForLayout($this->getLayoutNameForPage($page));
+    }
+
+    /**
+     * Returns full path for provided layout name
+     *
+     * @param string $layout layout name
+     *
+     * @return string layout full path
+     */
+    public function getLayoutRelativePathForLayout($layout)
```

Comment:
is the naming correct? sounds strange. getLayout{..}ForLayout :thinking: ?

Reply:
I was inspired by `getLayoutRelativePathForPage`, what should I name the function?

Reply:
looking at what it does, maybe it could be "getLayoutRelativePathByName" or just "getRelativePathForLayout"? Also maybe the param name could be more precise, like $layoutName. (just suggestions, but you understand better whats hapening here, maybe you could insipre from it and adapt the name accordingly)

Reply:
@zuk3975 @kpodemski Naming updated 👍 Ready for review